apidoc/package.json

```diff
@@ -6,6 +6,6 @@
     "apidoc": ">0.17.4"
   },
   "scripts": {
-    "apidoc": "apidoc -o ../src/main/webapp/docs -i ../ -f \"/src/main/groovy/.*\\.groovy$\" -f \"/apidoc/templates.*\\.js$\" "
+    "apidoc": "apidoc -o ../src/main/webapp/api/docs -i ../ -f \"/src/main/groovy/.*\\.groovy$\" -f \"/apidoc/templates.*\\.js$\" "
```

Comment:
fixes link from readme.md which has url for apiDocs at http://localhost:8080/api/docs